src/table/components/TablePaginationActions.jsx

```diff
@@ -72,44 +62,54 @@ export default function TablePaginationActions(props) {
   const handleLastButtonTab = keyboard.enabled ? (event) => handleLastTab(event, isInSelectionMode) : null;
 
   const getButton = (disabledCondition, pageNumber, type, onKeyDown = null) => {
-    const IconComponent = icons[type];
+    let iconType = type;
+    if (isRTL) {
+      if (type === 'FirstPage') {
+        iconType = 'LastPage';
+      } else if (type === 'PreviousPage') {
+        iconType = 'NextPage';
+      } else if (type === 'NextPage') {
+        iconType = 'PreviousPage';
+      } else if (type === 'LastPage') {
+        iconType = 'FirstPage';
+      }
+    }
```

Comment:
I would put this `icons` object. So you add `FirstPageRTL: LastPageIcon` and so on. The when you something like
```
iconType = `${type}${isRTL ? 'RTL' : ''}`
```

Reply:
isRTL is only used here so no need to create the constant

Reply:
I think I understand what you mean now. Sure. Done.